src/EditorFeatures/Core.Wpf/LineSeparators/LineSeparatorAdornmentManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Editor.Implementation.Adornments;
+using Microsoft.CodeAnalysis.Editor.Implementation.LineSeparators;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+
+namespace Microsoft.CodeAnalysis.Editor.LineSeparators
+{
+    internal class LineSeparatorAdornmentManager : AdornmentManager<LineSeparatorTag>
+    {
+        public LineSeparatorAdornmentManager(IThreadingContext threadingContext, IWpfTextView textView, IViewTagAggregatorFactoryService tagAggregatorFactoryService, IAsynchronousOperationListener asyncListener, string adornmentLayerName)
+            : base(threadingContext, textView, tagAggregatorFactoryService, asyncListener, adornmentLayerName)
+        {
+        }
+    }
```

Comment:
this doesn't appear to be using any of the protected members of the base class.  indeed, this type doesn't seem to have any useful logic/state in it at all.  i'm not sure why it needs to exist.

Reply:
Added the method that differentiates it from inline errors.